modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -20,11 +20,14 @@
   }
 
   .tox-promotion-link {
+    align-items: unsafe center;
```

Comment:
So it will start clipping to the height of the container if the emoji renders to high. Never used `unsafe` before?

Reply:
It won’t be clipping anything unless we set an `overflow` for the button. `unsafe` just prevents contents from stretching parent container but since the intended content here is a line of text it should be fine.

Reply:
But if it prevents the parent container from stretching what does it do if it doesn't clip scroll, overflow outside of the box?

Reply:
I'm just curious if say an emoji some how renders very large on some system since the default emoji rendering is different will what would happen.

Reply:
Yeah, it’s contents are gonna extend beyond the container. In that case on both top and bottom of the button, keeping text centered. 
I highly doubt that there is a system that has default emoji size almost twice the size of a text that it sits in, since emoji are intended to be used along the text. But yeah, if something goes really south then thunderbolt will be sticking out at the top and the bottom of the button.

Reply:
Right thanks for clarifying that.